src/main/java/io/stargate/sgv2/jsonapi/service/bridge/serializer/CustomValueSerializers.java

```diff
@@ -64,7 +64,7 @@ public static Map<QueryOuterClass.Value, QueryOuterClass.Value> getDoubleMapValu
   public static List<QueryOuterClass.Value> getDocumentIdValue(DocumentId documentId) {
     // Temporary implementation until we convert it to Tuple in DB
     List<QueryOuterClass.Value> tupleValues =
-        List.of(Values.of(documentId.typeId()), Values.of(documentId.toString()));
+        List.of(Values.of(documentId.typeId()), Values.of(documentId.asDBKey()));
```

Comment:
Necessary to avoid single-quotes from being added wrt `DocumentId.toString()` change: also semantically good to have separate method for use case.